controllers/operatorscaler_controller.go

```diff
@@ -144,6 +149,10 @@ func (r *OperatorScalerReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, err
 	}
 
+	if err := r.createMetrics(); err != nil {
```

Comment:
possible to stick to naming this as `reconcile...`?

Reply:
I can do that.

Reply:
Done